src/consensus/aft/raft.h

```diff
@@ -1008,12 +1014,13 @@ namespace aft
     {
       std::unique_lock<SpinLock> guard(state->lock);
       LOG_DEBUG_FMT(
-        "Received pt: {} pi: {} t: {} i: {} toi: {}",
+        "Received append entries: {}.{} to {}.{} (primary is {} in term {})",
```

Comment:
I think this rewording makes this log line much more readable. I'm tempted to do a general rewrite of all the logging in `raft.h` to make it more concise and consistent, but I'm wary of losing any information/patterns that other people rely on. In any case it likely remains incredibly spammy and verbose, so probably not worth doing.

Reply:
All the shorthand stuff is old and from the driver test, I like the view.seqno notation better.